.travis.yml

```
@@ -2,13 +2,15 @@ language: python
 dist: xenial
 jobs:
   include:
-    - python: '3.6'
-      script: pytest --cov=pysatMadrigal/
     - name: '3.7 with flake8'
       python: '3.7'
-      script: pytest --flake8 --cov=pysatMadrigal/
+      script:
+          - flake8
```

```suggestion
          - flake8 . --count --select=E,F,W
```
Flake8 will ignore warnings unless we tell it to error on these as well  (pytest-flake8 errors on warnings by default).